cosmic/FIRE.py

```diff
@@ -55,15 +55,17 @@ def SFH(n_samp):
     cwd = os.getcwd()
     data_dir = cwd+'/data/'
     FIRE = pd.read_hdf(data_dir+'FIRE.h5')
-
-    kde_dat = np.vstack([FIRE.age, FIRE.met])
-    SFH_kde = stats.gaussian_kde(kde_dat, bw_method=0.01)
-    dat_sample = SFH_kde.resample(n_samp)
-
+   
+    FIRE_cut = FIRE.loc[:,['age','met']]
+    #kde_dat = np.vstack([FIRE.age, FIRE.met])
+    #SFH_kde = stats.gaussian_kde(kde_dat, bw_method=0.01)
```

Comment:
@chiragchawla134  Can you delete the commented out code to keep things clean? Thanks!